applications/doppelganger-mirror/doppleganger.js

```diff
@@ -189,7 +189,8 @@ Doppleganger.prototype = {
             visible: false, // normally false
             modelURL: this.skeletonModelURL, //was field: url
             position: this.position,
-            rotation: this.orientation
+            rotation: this.orientation,
+            useOriginalPivot: true

```

Comment:
This would break on every avatar that doesn't have its pivot point in the right spot, won't it?

Reply:
Possible. I never saw an avatar with the reference point not being on the floor level. 
The thing is that this script was working fine before , someone mess with something and it's no more working.

this worked immediately for me and my avatars, including those from RPM.

That said, you can refuse this fix and ask to the person that break it to find why it doesn't work anymore.
for sure I'm going to use this one for my self until this happen. (I wonder when)

Reply:
I'd say that the assumption about coordinate system is very reasonable right now - almost all avatars are made this way. I think we should merge this for now because figuring out why original version is not working at the moment will be very difficult. What are your opinions?